.github/workflows/build-master-arm64.yaml

```diff
@@ -9,7 +9,7 @@ concurrency:
 jobs:
   build-green-arm64:
     runs-on: [self-hosted, arm64]
-    if: contains(["mudler", "davidcassany", "itxaka", kkaempf", "cOS-cibot"], "${{ github.actor }}") || contains(github.event.pull_request.labels.*.name, 'safe to test')
```

Comment:
how the F did this work?????

Reply:
id didnt work since the start lmao. The expression engine does not accept array literals but it just doesnt fail LMAO